authors/apps/profiles/serializers.py

```diff
@@ -8,7 +8,7 @@ class ProfileSerializer(serializers.ModelSerializer):
     class Meta:
         model = Profile
         fields = ("username", "first_name", "last_name", "bio", "image",
-                  "following",)
+                  "following", "date_of_birth")
```

Comment:
@pbkabali , Good job mate, but unless am wrong, seems like we are not supposed to return a user's DOB on their profile. The readme shows which fields to return.

Reply:
@codjoero, we had a discussion about this and agreed that It would be a great addition.

Reply:
Could you please then update the read me with this new informations

Reply:
I hear you @missvicki but had the discussion with the rest of the team.